src/ape/api/transactions.py

```diff
@@ -385,6 +385,17 @@ def return_value(self) -> Any:
 
         return output
 
+    @property
+    @raises_not_implemented
+    def traceback(self) -> Traceback:
+        """
+        A pythonic style traceback for both failing and non-failing receipts.
+        Requires a provider that implements
+        :meth:~ape.api.providers.ProviderAPI.get_transaction_trace`.
+        """
+
+        return  # type: ignore[return-value]
```

Comment:
Do you need a `return` statement w/ `raises_not_implemented`?

Reply:
If there is a return it needs something, but the other cases we do the type ignore in the signature without the return, which is better, so I will switch it to that

Reply:
done, but has to squash commits because of hefting rebasing (easier to rebase 1 commit), but the change is in there.